conf/eslint-recommended.js

```diff
@@ -6,274 +6,10 @@
-        "no-underscore-dangle": "off",
-        "no-unexpected-multiline": "error",
-        "no-unmodified-loop-condition": "off",
-        "no-unneeded-ternary": "off",
-        "no-unreachable": "error",
-        "no-unsafe-finally": "error",
-        "no-unsafe-negation": "error",
-        "no-unused-expressions": "off",
-        "no-unused-labels": "error",
-        "no-unused-vars": "error",
-        "no-use-before-define": "off",
-        "no-useless-call": "off",
-        "no-useless-catch": "off",
-        "no-useless-computed-key": "off",
-        "no-useless-concat": "off",
-        "no-useless-constructor": "off",
-        "no-useless-escape": "error",
-        "no-useless-rename": "off",
-        "no-useless-return": "off",
-        "no-var": "off",
-        "no-void": "off",
-        "no-warning-comments": "off",
-        "no-whitespace-before-property": "off",
-        "no-with": "off",
-        "nonblock-statement-body-position": "off",
-        "object-curly-newline": "off",
-        "object-curly-spacing": "off",
-        "object-property-newline": "off",
-        "object-shorthand": "off",
-        "one-var": "off",
-        "one-var-declaration-per-line": "off",
-        "operator-assignment": "off",
-        "operator-linebreak": "off",
-        "padded-blocks": "off",
-        "padding-line-between-statements": "off",
-        "prefer-arrow-callback": "off",
-        "prefer-const": "off",
-        "prefer-destructuring": "off",
-        "prefer-numeric-literals": "off",
-        "prefer-object-spread": "off",
-        "prefer-promise-reject-errors": "off",
-        "prefer-reflect": "off",
-        "prefer-rest-params": "off",
-        "prefer-spread": "off",
-        "prefer-template": "off",
-        "quote-props": "off",
-        quotes: "off",
-        radix: "off",
-        "require-atomic-updates": "off",
-        "require-await": "off",
-        "require-jsdoc": "off",
-        "require-unicode-regexp": "off",
-        "require-yield": "error",
-        "rest-spread-spacing": "off",
-        semi: "off",
-        "semi-spacing": "off",
-        "semi-style": "off",
-        "sort-imports": "off",
-        "sort-keys": "off",
-        "sort-vars": "off",
-        "space-before-blocks": "off",
-        "space-before-function-paren": "off",
-        "space-in-parens": "off",
-        "space-infix-ops": "off",
-        "space-unary-ops": "off",
-        "spaced-comment": "off",
-        strict: "off",
-        "switch-colon-spacing": "off",
-        "symbol-description": "off",
-        "template-curly-spacing": "off",
-        "template-tag-spacing": "off",
-        "unicode-bom": "off",
-        "use-isnan": "error",
-        "valid-jsdoc": "off",
-        "valid-typeof": "error",
-        "vars-on-top": "off",
-        "wrap-iife": "off",
-        "wrap-regex": "off",
-        "yield-star-spacing": "off",
-        yoda: "off"
-    }
+    rules: Object.assign({}, ...Object.keys(builtInRules).map(ruleId => ({
```

Comment:
Using `Array.prototype.reduce` can remove `Object.assign` and `Array.prototype.map`. And there is only one `{}` while using `Object.assign` needs many objects.